src/locale/locales/fr/messages.po

```diff
@@ -6804,6 +6368,11 @@ msgstr "Menu de mot-clé : {displayTag}"
 msgid "Tags only"
 msgstr "Mots-clés seulement"
 
+#: src/screens/Settings/AppIconSettings.tsx:42
+#: src/screens/Settings/AppIconSettings.tsx:76
+msgid "Tap to change app icon"
+msgstr "Tapper pour changer l’icône de l’application"
```

Comment:
unsure about parisian french or other variants of french, but "tapper" in canadian french is similar to "to punch". "**Appuyez**" would be a better fit

Reply:
`Tapper` in that sense in not unheard of on this side of the Atlantic ocean, but `Appuyez` is better anyway, so let's do that.